main.h

```diff
@@ -22,7 +22,7 @@
 
 #include <stdlib.h>
 #include <sys/time.h>
-#include <net/if.h>
+//#include <net/if.h>
```

Comment:
/usr/include/net/if.h:44:5: error: expected identifier before numeric constant
     IFF_UP = 0x1,  /* Interface is up.  */
     ^
In file included from /home/hankpeng/work/horst/main.h:25:0,
                 from /home/hankpeng/work/horst/ifctrl-wext.c:27:
/usr/include/net/if.h:111:8: error: redefinition of ‘struct ifmap’
 struct ifmap
        ^
In file included from /usr/include/linux/wireless.h:74:0,
                 from /home/hankpeng/work/horst/ifctrl-wext.c:25:
/usr/include/linux/if.h:143:8: note: originally defined here
 struct ifmap {